infrastructure/src/main/java/org/corfudb/infrastructure/CorfuServer.java

```diff
@@ -166,6 +167,9 @@ public class CorfuServer {
                     + "              The max size as percentage of underlying file-store size.\n "
                     + "              If this limit is exceeded "
                     + "              write requests will be rejected [default: 100.0].\n         "
+                    + " --orthogonal-data-kb=<orthogonal_data_kb>                                "
+                    + "              The partition offset that does not belong to its            "
+                    + "              intended use [default: 0].\n                              "
```

Comment:
I think offset is confusing here. This value is just "reserved space" for other files.

I think its just cleaner to just accept "corfu-reserved" and base the calculations off of that, instead of having to derive the value and do all this computation.